qa/tests/features/new/manual/wizard/wizard.feature

```diff
@@ -0,0 +1,139 @@
+@regression
+Feature: Main page, Wizard
+
+Background: 
+	Given user opened anfisa main page
+
+Scenario: 01 Open "Main Table" for WS dataset
+	When user clicks any WS dataset
+	And "View all variants" check-box is checked
+	And user clicks "Open" button 
+	Then the "Main Table" for chosen dataset should be opened
+
+Scenario: 02 Open "Filter Refiner" for WS dataset
+	When user clicks any WS dataset 
+	And checks "Explore data or build new filter" check-box in the right down corner
+	And clicks "Open" button
+	Then the "Filter Refiner" for chosen dataset should be opened
+
+Scenario: 03 Open "Main Table" for derived dataset
+	When user clicks any dataset with derived datasets 
+	And checks "Use an existing candidate set" check-box
+	And clicks "Continue" button
+	And chooses derived dataset 
+	And clicks "Open" button
+	Then the "Main Table" for chosen dataset should be opened
+
+Scenario: 04 "You previously worked with" field 
+	 When user clicks any dataset 
+	 Then the information about the chosen dataset should be displayed
+	 And "You previously worked with" field should be displayed in the upper right part of the screen
+	 And previously opened datasets should be written 
+
+Scenario: 05 "Edit choice" button
+	When user clicks any XL dataset
+	And "Whole genome/exome" check-box is checked
+	And clicks "Continue" button
+	And user clicks "Edit choice" button along "Whole genome/exome" check-box
+	Then second window should be closed
+	And user should be able to change the choice
+
+Scenario: 06 Add description to WS dataset
+	When user clicks any WS dataset
+	And clicks "Pencil" sign near the "Description"
+	And writes any description
+	Then the description should be saved in couple of seconds
+	And message about description saving should be shown
+
+Scenario: 07 Open "Filter Refiner" with additional preset filter
+	When user clicks "PGP3140_wgs_panel_hl" dataset
+	And checks "Apply additional preset filter" check-box
+	And chooses any preset before "@BGM Research" from "Additional preset filters" list
+	And clicks "Open" button
+	Then the "Filter Refiner" for chosen filter should be opened
+
+Scenario: 08 Open "Decision Tree" with additional preset filter
+	When user clicks "PGP3140_wgs_panel_hl" dataset
+	And checks "Apply additional preset filter" check-box
+	And chooses any preset after "@BGM Research" from "Additional preset filters" list
+	And clicks "Open" button
+	Then the "Decision Tree Panel" for chosen filter should be opened
+
+Scenario: 09 "What's next?" window
```

Comment:
For scenarios 10-13 you also need to add check that user can open a custom scenario with such rubric